src/interfaces/IBFactory.sol

```diff
@@ -4,17 +4,48 @@ pragma solidity 0.8.23;
 import {IBPool} from 'interfaces/IBPool.sol';
 
 interface IBFactory {
+  /**
+   * @notice Emitted when creating a new pool
+   * @param caller The caller of the new pool function
+   * @param pool The address of the new pool
+   */
   event LOG_NEW_POOL(address indexed caller, address indexed pool);
 
+  /**
+   * @notice Emitted when setting the BLabs address
+   * @param caller The caller of the set BLabs function
+   * @param bLabs The address of the new BLabs
+   */
   event LOG_BLABS(address indexed caller, address indexed bLabs);
 
+  /**
+   * @notice Creates a new BPool, assigning msg.sender as the controller
+   * @return pool The new BPool
+   */
   function newBPool() external returns (IBPool pool);
 
+  /**
+   * @notice Sets the BLabs address in the factory
+   * @param b The new BLabs address
+   */
   function setBLabs(address b) external;
 
+  /**
+   * @notice Collects the balance of a pool and transfers it to BLabs address
```

Comment:
hmmmm i'd be coherent with the next description and talk about fees, not "balance of a pool"